apps/web/src/common/constants.ts

```diff
@@ -37,3 +37,8 @@ export const PlanningSteps = [
   'Activities Gap',
   'Suggestions',
 ];
+
+export enum TagStyles {
```

Comment:
There're tags for Care activities as well. I guess we can be more specific of what tags these are